client/components/post-card/small.jsx

```diff
@@ -15,22 +15,10 @@ import safeImageUrl from 'lib/safe-image-url';
 import resizeImageUrl from 'lib/resize-image-url';
 import AuthorAndSite from './author-and-site';
 
-export function SmallPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
+export function SmallPostCard( { post, site, onPostClick = noop, onSiteClick = noop } ) { // eslint-disable-line no-unused-vars
```

Comment:
Do we need both removing `translate` and `// eslint-disable-line no-unused-vars`?